rtlive/sources/data_it.py

```diff
@@ -1,39 +1,210 @@
+import io
 import logging
 import pandas
+import requests
+
+from datetime import datetime, timedelta
 
 from . import ourworldindata
 from .. import preprocessing
 
 _log = logging.getLogger(__file__)
 
+IT_REGION_NAMES = {
+    '1': 'Piemonte',
+    '2': 'Valle d\'Aosta',
+    '3': 'Lombardia',
+    '5': 'Veneto',
```

Comment:
I don't know anything about this, but this is going from 3 to 5 directly -- is it expected?

Reply:
Ran into this and I think I can answer, even if I am not the original author (great job @davideferre!). Region 4 is Trentino/Alto Adige which is a region made by two autonomous provinces that actually behave like regions in many ways. The two provinces have the assigned codes 21 and 22 and they are reported like that in the data (see e.g. https://github.com/pcm-dpc/COVID-19/blob/master/dati-regioni/dpc-covid19-ita-regioni-20200224.csv).

Reply:
Yes thank you @pietroppeter!